README.md

```diff
@@ -111,7 +107,7 @@ environment.
 | `LOG_LEVEL`                        | [log level](https://github.com/trentm/node-bunyan#levels)                                                   | Depends on `NODE_ENV`  |          |
 | `NODE_ENV`                         | node environment                                                                                            | development            |          |
 | `SYNDICATION_API_KEY`              | API key to access syndication                                                                               |                        | yes      |
-| `API_URL`                          | URL to Syndicate service to scrape                                                                          | http://v1.syndication.nhschoices.nhs.uk/services/types/sexualhealthinformationandsupport | no      |
+| `SYNDICATION_SERVICE_END_POINT`    | URL to Syndicate service to scrape                                                                          | http://v1.syndication.nhschoices.nhs.uk/services/types/sexualhealthinformationandsupport | no      |
 | `OUTPUT_FILE`                      | Filename saved to azure                                                                                     | shis-data              | no       |
```

Comment:
I've commented on this separately for the reference in `docker-compose` to allow the file to name to be overridden otherwise it isn't really an env var in the same way the others are.